Tests/AlicerceTests/UIKit/UIColorTestCase.swift

```diff
@@ -38,4 +38,26 @@ final class UIColorTestCase: XCTestCase {
         XCTAssertEqual(ciColor.blue, 1.0)
         XCTAssertEqual(ciColor.alpha, 0.0)
     }
+
+    func testHexString_WithBaseColors_ItShouldReturnCorrectHexColor() {
+
+        let hexRed = UIColor.red.hexString
+        let hexGreen = UIColor.green.hexString
+        let hexBlue = UIColor.blue.hexString
+
+        XCTAssertEqual(hexRed, "#ff0000")
+        XCTAssertEqual(hexGreen, "#00ff00")
+        XCTAssertEqual(hexBlue, "#0000ff")
+    }
+
+    func testHexString_WithAlphaColors_ItShouldReturnCorrectHexColor() {
+
+        let hexOpaque = UIColor(white: 1.0, alpha: 1.0).hexStringWithAlpha
+        let hexSemi = UIColor(white: 1.0, alpha: 0.5).hexStringWithAlpha
+        let hexTransparent = UIColor(white: 1.0, alpha: 0.0).hexStringWithAlpha
+
+        XCTAssertEqual(hexOpaque, "#ffffffff")
+        XCTAssertEqual(hexSemi, "#7fffffff")
+        XCTAssertEqual(hexTransparent, "#00ffffff")
+    }
 }
```

Comment:
following the above comments regarding the hex format, I would suggest we add some tests encoding  <-> decoding using our own API's and validating that the colors effectively match 😄